frontend/src/components/Shared/Authcommon/AuthForm.jsx

```
@@ -67,68 +96,64 @@ const AuthForm = ({
+                      cachedGoogleAccount.photoURL ? (
+                        <img 
+                          src={cachedGoogleAccount.photoURL} 
+                          alt="Profile" 
+                          className="w-4 h-4 sm:w-5 sm:h-5 rounded-full" 
+                        />
+                      ) : (
+                        <svg 
+                          xmlns="http://www.w3.org/2000/svg" 
+                          viewBox="0 0 24 24" 
+                          className="w-4 h-4 sm:w-5 sm:h-5 fill-current"
+                        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)